### PR TITLE
Update stable weights for Weights 2.0

### LINKED ISSUE #1606: Capacity Stable Weights Update PoV sizes
The PoV weights for the stable weights are not all set. These need to be updated with the correct PoV weights.

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to update the stable weights with the PoV and Weights 2.0.

This PR takes the approach of just completely updating the weights, so the stable weights DO CHANGE by more than just the addition of the PoV data.

I am open to a alternative approach that *only* adds the PoV, but I judged that any change might as well just be a complete update.

Closes #1606

### CODE REVIEW

File: pallets/frequency-tx-payment/src/capacity_stable_weights.rs [hunk 1]
```diff
@@ -80,42 +80,74 @@ pub trait WeightInfo {
...
+		//  Estimated: `14946`
+		// Minimum execution time: 115_100_000 picoseconds.
+		Weight::from_parts(120_342_076, 14946)
+			// Standard Error: 35_029
+			.saturating_add(Weight::from_parts(34_990, 0).saturating_mul(s.into()))
+			.saturating_add(T::DbWeight::get().reads(8_u64))
+			.saturating_add(T::DbWeight::get().writes(4_u64))
 	}
 	// Storage: Msa PublicKeyToMsaId (r:1 w:0)
 	// Storage: Msa DelegatorAndProviderToDelegation (r:1 w:1)
```
Comment (wilwade) [inline pallets/frequency-tx-payment/src/capacity_stable_weights.rs:153]: `grant_schema_permissions` below is removed in #1661

File: pallets/frequency-tx-payment/src/capacity_stable_weights.rs [hunk 2]
```diff
@@ -80,42 +80,74 @@ pub trait WeightInfo {
...
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	// Storage: Msa PayloadSignatureBucketCount (r:1 w:1)
-	// Storage: Msa PayloadSignatureRegistry (r:1 w:1)
-	// Storage: Msa PublicKeyToMsaId (r:2 w:1)
-	// Storage: Msa ProviderToRegistryEntry (r:1 w:0)
-	// Storage: Msa CurrentMsaIdentifierMaximum (r:1 w:1)
-	// Storage: Msa PublicKeyCountForMsaId (r:1 w:1)
-	// Storage: Msa DelegatorAndProviderToDelegation (r:1 w:1)
-	// Storage: Schemas CurrentSchemaIdentifierMaximum (r:1 w:0)
+	/// Storage: Msa PayloadSignatureRegistryList (r:2 w:2)
```
Comment (wilwade) [inline pallets/frequency-tx-payment/src/capacity_stable_weights.rs:83]: These updated weights are just copied out of the current `weights.rs` files

File: pallets/frequency-tx-payment/src/tests/stable_weights_tests.rs [hunk 3]
```diff
@@ -7,22 +7,63 @@ use crate::capacity_stable_weights::{SubstrateWeight, WeightInfo};
...
-			(SubstrateWeight::<Test>::add_onchain_message(100), 139576386),
-			(SubstrateWeight::<Test>::add_ipfs_message(), 131669000),
-			(SubstrateWeight::<Test>::apply_item_actions(100), 66240801),
-			(SubstrateWeight::<Test>::upsert_page(100), 23063086),
-			(SubstrateWeight::<Test>::delete_page(), 26000000),
-			(SubstrateWeight::<Test>::apply_item_actions_with_signature(100), 106536191),
-			(SubstrateWeight::<Test>::upsert_page_with_signature(100), 61765307),
-			(SubstrateWeight::<Test>::delete_page_with_signature(), 65000000),
-			(SubstrateWeight::<Test>::claim_handle(100), 100989953),
+			(
```
Comment (enddynayn) [inline pallets/frequency-tx-payment/src/tests/stable_weights_tests.rs:10]: nice

**Review by enddynayn — APPROVED** [review]
👍 fabulous!

**Review by saraswatpuneet — APPROVED** [review]

**Review by saraswatpuneet — APPROVED** [review]
sampled a few weights and matched with 1.7.4
so looks good :+1:

**Pull request merged**